src/CSSList/CSSList.php

```diff
@@ -41,7 +41,7 @@ abstract class CSSList implements Renderable, Commentable
     protected $comments = [];
 
     /**
-     * @var array<int<0, max>, RuleSet|CSSList|Import|Charset>
+     * @var array<int<0, max>, RuleSet|CSSList|DeclarationBlock|Import|Charset>
```

Comment:
I propose we introduce an interface instead, e.g. something like `CSSListItem`. (We could do this as a pre-patch to keep PRs small.)

Reply:
Doing so would also make Stan happy :)